Analysis/include/Luau/Type.h

```diff
@@ -499,29 +499,32 @@ struct ClassType
     Tags tags;
     std::shared_ptr<ClassUserData> userData;
     ModuleName definitionModuleName;
+    Location definitionLocation;
```

Comment:
Would be better as `std::optional<Location>` since many definitions do not have an assigned location and it's more consistent with other optional location fields.

Reply:
Makes sense, I copied this from `TableType` where it is non-optional